pkg/volume/rbd/rbd.go

```diff
@@ -239,3 +242,67 @@ func (plugin *rbdPlugin) execCommand(command string, args []string) ([]byte, err
 	cmd := plugin.exe.Command(command, args...)
 	return cmd.CombinedOutput()
 }
+
+func (plugin *rbdPlugin) runInSidecarContainer(containerName string, cmd []string, args []string) ([]byte, error) {
+	priv := true
+	pod := &api.Pod{
+		ObjectMeta: api.ObjectMeta{
+			GenerateName: "rbd-sidecar-",
+			Namespace:    api.NamespaceDefault,
+		},
+		Spec: api.PodSpec{
```

Comment:
Don't you need to set the `NodeName` in the pod soec to ensure this pod runs on the same node?